data/data/rhcos.json

```diff
@@ -1,104 +1,111 @@
 {
     "amis": {
         "ap-northeast-1": {
-            "hvm": "ami-0eae1951134909fc5"
+            "hvm": "ami-0db1f56b7e43c2dcf"
         },
         "ap-northeast-2": {
-            "hvm": "ami-0b48a35eaf507815f"
+            "hvm": "ami-037ba98a635353cd5"
         },
         "ap-south-1": {
-            "hvm": "ami-032e831b98004ca91"
+            "hvm": "ami-06ca6319f9a7cc7ef"
         },
         "ap-southeast-1": {
-            "hvm": "ami-08fd6ebd44652434c"
+            "hvm": "ami-09dc67bc03673a533"
         },
         "ap-southeast-2": {
-            "hvm": "ami-05add5fb4ca2885d0"
+            "hvm": "ami-0005fdca8c2b639cd"
         },
         "ca-central-1": {
-            "hvm": "ami-0c09b9aa5936ee7eb"
+            "hvm": "ami-0937ba51d6915aae8"
         },
         "eu-central-1": {
-            "hvm": "ami-033d5e7102da60fef"
+            "hvm": "ami-0060b31e9ef744e3d"
         },
         "eu-west-1": {
-            "hvm": "ami-0790e45db9ab322e4"
+            "hvm": "ami-008aa9cb0e9723ae7"
         },
         "eu-west-2": {
-            "hvm": "ami-04178618392bedbd7"
+            "hvm": "ami-02ff9f8182d312adc"
         },
         "eu-west-3": {
-            "hvm": "ami-09441e9d1eee0ac00"
+            "hvm": "ami-064c1a19b5600d4bb"
         },
         "sa-east-1": {
-            "hvm": "ami-045f1c2856ec20f5a"
+            "hvm": "ami-0ee0e369b4244858d"
         },
         "us-east-1": {
-            "hvm": "ami-0df3f99538fbef10f"
+            "hvm": "ami-033b8fc59b43c23ee"
         },
         "us-east-2": {
-            "hvm": "ami-098d8cd80abb37c73"
+            "hvm": "ami-02200f690a88f0819"
         },
         "us-west-1": {
-            "hvm": "ami-0e2b5d1dcfbcc0c8f"
+            "hvm": "ami-03fc8933dc66db8f0"
         },
         "us-west-2": {
-            "hvm": "ami-02b30dab0946cf89d"
+            "hvm": "ami-096cb19297a5c2726"
         }
     },
-    "baseURI": "https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/ootpa/410.8.20190425.0/",
-    "buildid": "410.8.20190425.0",
+    "baseURI": "https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/ootpa/410.8.20190508.1/",
+    "buildid": "410.8.20190508.1",
     "images": {
+        "aws": {
```

Comment:
Hmm I feel like we removed this from the artifacts at some point, not sure how it got added back in, either case shouldn't affect anything

Reply:
I'll remove it.

Reply:
Removed